common/archive/demultiplexer.go

```diff
@@ -217,6 +230,11 @@ func (demux *Demultiplexer) BodyBSON(buf []byte) error {
 		return newError("collection data without a collection header")
 	}
 
+	// For atlas proxy archive restores, ignore collections from the admin DB.
+	if demux.IsAtlasProxy && strings.Contains(demux.currentNamespace, "admin") {
+		return nil
+	}
+
```

Comment:
it seems this line also ignores all collections containing "admin", is it really intended ?

Reply:
@rohshar @tfogo @autarch